packages/plugin-ngrok/src/tunnel.ts

```diff
@@ -9,9 +9,21 @@ import {outputToken, outputInfo, outputContent} from '@shopify/cli-kit/node/outp
 
 export default startTunnel({provider: TUNNEL_PROVIDER, action: hookStart})
 
+let cachedPort: number
```

Comment:
Introducing state in module is often a recipe for issues because we start developing logic on the assumption there's a single module in the graph, but that assumption might not hold true at some point. If the `dev` logic interacts with plugins to start, stop, and get the status, I think the state should be kept there and passed as needed.

Reply:
true, removed this cachedPort as there isn't really a need for it